debezium-embedded/src/test/java/io/debezium/embedded/AbstractConnectorTest.java

```diff
@@ -353,7 +354,7 @@ public void taskStarted() {
             engine.run();
         });
         try {
-            if (!latch.await(1000, TimeUnit.SECONDS)) {
+            if (!latch.await(5, TimeUnit.MINUTES)) {
```

Comment:
unrelated change (1000 seconds is about 16,6 minutes) but I think we don't have to wait so long? wdyt?